Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -242,6 +242,83 @@ class PlaceDetailViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
 
+    func openCard(forPlace place: Place) {
+        if currentCardViewController.place == place {
+            currentCardViewController.place = place
+            // if we're currently looking at that card then just animate in the event
+            currentCardViewController.showEvent()
+        } else {
+            // I'm totally not sure what we should be doing here.
```

Comment:
nit: how should we track this? (if not fixed in a later commit).

Is it that issue Ricardo is working on? Maybe list that here.